tutorials/pregnancy.md

```diff
@@ -8,3 +8,19 @@ Welcome to the pregnancy module. This code deals with fertility, pregnancy, ante
 - `src/hivpy/pregnancy_data.py` - A class for storing data loaded from `pregnancy.yaml`.
 
 If there are any pregnancy-related variables you would like to change before running your simulation, please change them in `pregnancy.yaml`.
+
+### Pregnancy Data Variables
+
+- *`can_be_pregnant`* - The proportion of women in a population that can become pregnant.
+- *`rate_want_no_children`* - The rate at which women stop wanting children over time.
+- *`date_pmtct`* - The year during which PMTCT begins (2004 by default).
+- *`pmtct_inc_rate`* - The rate at which the probability of receiving PMTCT increases per year.
+- *`fold_preg`* - A fertility factor that changes with age and affects pregnancy probability.
+- *`inc_cat`* - A demographic variable that affects population growth and pregnancy probability.
+- *`prob_pregnancy_base`* - The base probability of pregnancy (calculated when the pregnancy module is initialised).
+- *`rate_testanc_inc`* - An additive modifier to `prob_anc`.
```

Comment:
*`rate_testanc_inc`* - The amount by which `prob_anc` increases per timestep (additive)